### DISCUSSION TITLE
Prism.Avalonia for v11.0 Preview 5 NuGet Published!

### DISCUSSION THREAD

**Discussion by DamianSuess** [AvaloniaCommunity/Prism.Avalonia#48] [Announcements]
Hello everyone!

We are pleased to announce that support for Avaonia v11 Preview 5 has been published to NuGet.org

| Package | Download
|-|-|
| Prism.Avalonia | [![Prism.Avalonia NuGet Badge](https://buildstats.info/nuget/Prism.Avalonia?dWidth=70&includePreReleases=true)](https://www.nuget.org/packages/Prism.Avalonia/8.1.97.4-preview.11.5)
| Prism.DryIoc.Avalonia | [![Prism.DryIoc.Avalonia NuGet Badge](https://buildstats.info/nuget/Prism.DryIoc.Avalonia?dWidth=70&includePreReleases=true)](https://www.nuget.org/packages/Prism.DryIoc.Avalonia/8.1.97.4-preview.11.5)

As per the project status post #44, this was a [BIG update from Prev 4](https://github.com/AvaloniaUI/Avalonia/compare/release/11.0.0-preview4...release/11.0.0-preview5), and you should expect some breaking changes that need to be carefully taken care of. Before posting a new Issue or Discussion, please review Avalonia's official [Breaking Changes](https://github.com/AvaloniaUI/Avalonia/wiki/Breaking-Changes) wiki page as well as our [Upgrading to Avalonia v11.md](https://github.com/AvaloniaCommunity/Prism.Avalonia/blob/develop/Upgrading-to-Avalonia-11.md) document.

## Pull Requests

* #39 - Upgrade to Avalonia v11.0 Preview 5
* #42 - Update Windows Notification Manager Sample
* #45 - Rename XAML to AXAML file extension
* #46 - Theme Fixes
    * Upgrade to the new theme system
    * Provides a sample of theme switching
* #47 - After Upgrading Prism Module Region Fails Inside of ItemsControl
    * Fix ItemsControlRegionAdapter after upgrading

## Release Roadmap

* [x] Upgrade system to support v11 Preview 5 (PR #39)
* [x] Document [Upgrading and Breaking Changes](https://github.com/AvaloniaCommunity/Prism.Avalonia/blob/develop/Upgrading-to-Avalonia-11.md))
* [x] Repair Region registration after upgrading
* [x] Validate unit tests 
    * [x] Document changes required
* [x] Update Sample Applications
    * [x] New Theme System
    * [x] Theme switching example
    * [x] Window Notification Manager (PR #42)
    * [x] Fix BootstrapperAppExample (_v11 breaking changes_)
    * [x] Fix ModulesSample app (_v11 breaking changes_) (PR #47)
    * [x] Fix the sample, ViewDiscovery. Broken since v11 Preview-1.

## Stretch Goals

* [ ] IDialogService allow for callback to be optional

## Future Roadmap

* Visual Studio Template Extension
* Code generator for data bindings and extensions (separate '_toolkit_' package)

**Comment by OmidID** [comment]
Great job 👍

**Comment by Coolkeke** [comment]
Very good, strong support 👍
